src/Compiler/AbstractIL/il.fs

```diff
@@ -2865,6 +2905,10 @@ type ILTypeDef
     member x.WithInitSemantics(init) =
         x.With(attributes = (x.Attributes ||| convertInitSemantics init))
 
+    member x.WithAdditionalFlags(flags, rewriteInsteadOfAdd) =
```

Comment:
Can you please turn it into two distinct members instead of a bool flag? This helps the consuming side.

x.WithReplacedFlags()
x.WithAdditionalFlags()


The semantics of .WithAdditionalflags(..,true) feels wrong, since it is not additional.

Reply:
I think it will be even better to make specialized methods for each group of flags instead of a general method that requires an additional thinking about the correct use
![image](https://github.com/user-attachments/assets/90edf2b7-e9a8-4bfa-8dfd-8ff1906d4135)